_Group Projects/Group_5/Week12_Assignment/README.md

```diff
@@ -0,0 +1,28 @@
+## Group 5 Week 12 Assignment
+## Noah's First Draft
+
+### Dataset
+
+We simulated a very simple dataset. We have 100 enumerators, each of whom asks a respondent for his/her name (coded as respondent_id), phone number, if he/she consents to the survey, and his/her age. The dataset also records the survey date, the survey start time, the start time of the consent question, the end time of that question, and the survey end time.
+
+### Check 1: Duplicates
+
+#### Potential Issue
+
+One common issue with data collection is that respondents are recorded multiple times in the dataset. Such duplication may be caused by enumerator error. Another possibility is that respondent(s) knowingly respond to multiple enumerators for additional compensation. We identify two issues with such duplication. First, duplicate observations may diminish our dataset's representativeness of the target population. Second, respondents may give contradictory answers in separate survey administrations, which may diminish the accuracy of our dataset.
+
+#### Check 1.1: Respondent ID
+
+This check compares the respondent_id, a stand-in for name, of each survey administration (i.e., row). The data associated with these duplicates are exported to the "Respondent ID Duplicates" sheet of the output Excel file. We encourage the field manager to examine these duplicates and discuss his/her findings with the appropriate enumerators.
+
+#### Check 1.2: Respondent Phone
+
+This check is identical to the one above, except it identifies duplicates using respondent phone numbers. Ali explained in class that respondents may give false names or data when trying to "game the system," but they often give real phone numbers. Accordingly, we have implemented this check.
```

Comment:
It can be helpful to include some pre-emptive measures to alleviate the error by verifying their phone number before the survey such as calling or sending a text message.

Reply:
It would be helpful if you could incorporate this point and Ali's feedback into the final product. Thank you!